src/constants/wallets.ts

```diff
@@ -163,6 +162,7 @@ export const wallets: Record<WalletType, WalletConfig> = {
     connectionTypes: [
       WalletConnectionType.CoinbaseWalletSdk,
       WalletConnectionType.InjectedEip1193,
+      WalletConnectionType.WalletConnect2,
       WalletConnectionType.WalletConnect1,
```

Comment:
i think we can just fully remove support for walletconnect1.. so removing the connection type, `walletconnect1Name` in options